src/execution/execute.ts

```diff
@@ -504,10 +514,37 @@ export function buildExecutionContext(
     typeResolver: typeResolver ?? defaultTypeResolver,
     subscribeFieldResolver: subscribeFieldResolver ?? defaultFieldResolver,
     subsequentPayloads: new Set(),
+    executionController: new ExecutionController(signal),
     errors: [],
   };
 }
 
+class ExecutionController {
+  /** For performance reason we can't use `signal.isAborted` so we cache it here */
+  isAborted: boolean = false;
+
+  private readonly _passedInAbortSignal: AbortSignal | undefined;
+  private readonly _abortController: AbortController | undefined =
```

Comment:
Same as about `AbortSignal` - I believe it was causing an error in node@14 so I had to add the interfaces that match `AbortSignal` and `AbortController`

Reply:
I will check on node@14. Maybe my assumption about TS detecting this type on node@14 is wrong.